### DISCUSSION TITLE
Vanced Manager 2.6.2

### DISCUSSION THREAD

**Discussion by KevinX8** [TeamVanced/VancedManager#709] [General]
- Fixed root installer for some users
- App now checks for Magisk and displays an error message if it's not detected
- Various optimizations

<hr /><em>This discussion was created from the release <a href='https://github.com/YTVanced/VancedManager/releases/tag/v2.6.2'>Vanced Manager 2.6.2</a>.</em>

**Comment by Wolfina2022** [comment]
There is a bug with youtube vanced its the notifivation spamming non stop

**Reply by Wolfina2022** [reply]
Same notifications

**Comment by MeMoNoVo** [comment]
Studio youtube vanced 2022/02/13

**Comment by MeMoNoVo** [comment]
Vanced studio youtube 2.6.2. Apk

**Reply by Wolfina2022** [reply]
I have that apk and ita doing that notification glich

**Reply by Wolfina2022** [reply]
I set my updater to latest version
